.github/CODEOWNERS

```diff
@@ -1,8 +1,6 @@
-# Team Lead
-* @Tuubz
-
 # Team Members
-* @AVI5HEK @chornerman @doannimble @hoangnguyen92dn @kaungkhantsoe @luongvo @lydiasama @manh-t @minhnimble @ryan-conway @sleepylee @Wadeewee 
+# @Tuubz is the Team Lead
+* @Tuubz @AVI5HEK @chornerman @doannimble @hoangnguyen92dn @kaungkhantsoe @luongvo @lydiasama @manh-t @minhnimble @ryan-conway @sleepylee @Wadeewee 
```

Comment:
Should we put `@Tuubz` in between `@sleepylee` & `@Wadeewee` to keep the alphabetical order? Since we already specify the TL as a comment: `@Tuubz is the Team Lead`

Reply:
@Tuubz it's up to you, please help me correct it